rai/node/rpc.cpp

```diff
@@ -4436,6 +4494,14 @@ void rai::rpc_handler::process_request ()
 		{
 			search_pending_all ();
 		}
+		else if (action == "request_reconfirmation")
```

Comment:
Should we call it "request_confirmation" instead?  I know that the protocol might make this look like a "reconfirmation" but RPC clients don't necessarily understand that part.

Reply:
Sure.